### PR TITLE
update the docs for rearranged args

### PR DESCRIPTION
**Pull request by adamspofford-dfinity**
dfinity/sdk#2369 moved middle-of-the-command named arguments to the end of the command. This updates the docs to reflect the new CLI interface. Do not merge before dfx 0.12

### CODE REVIEW

File: docs/developer-docs/build/backend/access-control.md [hunk 1]
```diff
@@ -197,15 +197,15 @@ To assign the admin role to the IC\_admin identity:
 
 2.  Assign the `ic_admin` principal the `admin` role by running a command similar to the following using Candid syntax:
 
-        dfx canister --wallet=$(dfx identity get-wallet) call access_hello assign_role '((principal "c5wa6-3irl7-tuxuo-4vtyw-xsnhw-rv2a6-vcmdz-bzkca-vejmd-327zo-wae"),opt variant{admin})'
+        dfx canister call --wallet="$(dfx identity get-wallet)" access_hello assign_role '((principal "c5wa6-3irl7-tuxuo-4vtyw-xsnhw-rv2a6-vcmdz-bzkca-vejmd-327zo-wae"),opt variant{admin})'
```
Comment (ericswanson-dfinity) [inline docs/developer-docs/build/backend/access-control.md:200]:
```suggestion
        dfx canister call access_hello assign_role '((principal "c5wa6-3irl7-tuxuo-4vtyw-xsnhw-rv2a6-vcmdz-bzkca-vejmd-327zo-wae"),opt variant{admin})' --wallet="$(dfx identity get-wallet)" 
```

File: docs/developer-docs/build/cdks/cdk-rs-dfinity/rust-profile.md [hunk 2]
```diff
@@ -263,7 +263,7 @@ To add identities for testing:
 2.  Call the `update` function to add a profile for the new identity. Enter your passphrase when prompted.
 
     ``` bash
-    dfx --identity Miles canister call rust_profile_backend update '(record {name = "Miles"; description = "Great Dane"; keywords = vec {"Boston"; "mantle"; "three-legged"}})'
+    dfx canister call --identity Miles rust_profile_backend update '(record {name = "Miles"; description = "Great Dane"; keywords = vec {"Boston"; "mantle"; "three-legged"}})'
```
Comment (ericswanson-dfinity) [inline docs/developer-docs/build/cdks/cdk-rs-dfinity/rust-profile.md:266]:
```suggestion
    dfx canister call rust_profile_backend update '(record {name = "Miles"; description = "Great Dane"; keywords = vec {"Boston"; "mantle"; "three-legged"}})' --identity Miles 
```

File: docs/developer-docs/build/cdks/cdk-rs-dfinity/rust-profile.md [hunk 3]
```diff
@@ -285,7 +285,7 @@ To add identities for testing:
 4.  Call the `getSelf` function using the `Miles` user identity by running the following command:
 
     ``` bash
-    dfx --identity Miles canister call rust_profile_backend getSelf
+    dfx canister call --identity Miles rust_profile_backend getSelf
```
Comment (ericswanson-dfinity) [inline docs/developer-docs/build/cdks/cdk-rs-dfinity/rust-profile.md:288]:
```suggestion
    dfx canister call rust_profile_backend getSelf --identity Miles
```

**Review by ericswanson-dfinity — CHANGES_REQUESTED** [review]
Just a couple spots left where `--network` can go after all of the positionals

**Comment by jessiemongeon1** [comment]
This PR appears to be outdated/not needed. Closing this PR to clean up the repo. Please feel free to reopen if you think this should be revisited.

**Pull request closed without merging**